reporter/html.js

```diff
@@ -365,6 +366,15 @@ export function escapeText( s ) {
 			dropDownList = document.createElement( "ul" ),
 			dirty = false;
 
+		const options = {
+			includeScore: true,
```

Comment:
I don't see score being used. Is this needed?

Reply:
Removed.